### DISCUSSION TITLE
支持redshift

### DISCUSSION THREAD

**Discussion by zxgangandy** [hhyo/Archery#2525] [Ideas]
请问目前的版本支持aws的redshift吗？如果不支持，未来是否有计划呢？

**Comment by LeoQuote** [comment]
你好, 本项目是一个开源项目, 你可以自由提功能需求, 但其实比提功能需求更加务实, 有价值的是实现这个功能, 本项目开发文档如下, 希望对你有帮助:

* https://github.com/hhyo/Archery/wiki/development
* https://github.com/hhyo/Archery/wiki/notifier

期待你的pr, 谢谢~
